examples/proxy/proxy_test.go

```diff
@@ -90,11 +90,11 @@ func (n *ProxyPlugin) ProxyBroadcast(node *network.Network, sender peer.ID, msg
 	return nil
 }
 
-// ExampleProxy demonstrates how to send a message to nodes which do not directly have connections
+// ProxyExample demonstrates how to send a message to nodes which do not directly have connections
 // to their desired messaging target.
 //
 // Messages are proxied to closer nodes using the Kademlia routing table.
-func ExampleProxy() {
+func ProxyExample() {
```

Comment:
Best keep the prefix as `Example` to stick to Go's recommendations for naming.

Reply:
go vet is assuming XXXExample with an identifier named "XXX".
actually the recommendation makes this naming not good. it is a demonstration of something but not an example of an identifier.

Reply:
https://blog.golang.org/examples That's where we got the recommendation from.

Reply:
um ... `ExampleProxyPlugin` could handle both cases.

Reply:
Sure, sounds good 👍 Also need to rename `BasicExample` as well 😄.

Reply:
done~